santosfamilyfoundation/RuinaMain
# [RUIN-221] Clean up testing tooltips

# Description

Added tooltips to the quickSurvey. Replaced function calls with the component. Updated quetions.js with newest tooltips.

# Testing

1. Start a new report
2. Check that tooltips appear for ownership and secondary crash
3. Continue to form page
4. Check that the tooltips are still appearing on the forms.
